Missing form types (#1719)
PrestaShop/docs: development/components/form/types-reference/typeahead-customer-collection-type.md

---
title: TypeaheadCustomerCollectionType
---

# TypeaheadCustomerCollectionType

> This form class is responsible to create a customer.

Reviewer: ```suggestion
This form class is responsible for selecting a customer.
```

creating or selecting?
